apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/SamhsaMatcherR4FromClaimTransformerV2Test.java

```diff
@@ -168,12 +168,9 @@ public static Stream<Arguments> data() throws IOException {
   @MethodSource("data")
   public void testR4SamhsaMatcherWhenTransformedClaimHasItemWithHcpcsCodeAndMatchingCptExpectMatch(
       String claimType, ExplanationOfBenefit loadedExplanationOfBenefit) {
-    boolean expectMatch = true;
+    boolean expectMatch = !PART_D_EVENT_CLAIM.equals(claimType);
 
-    // PDE has no SAMHSA, so expect no match on SAMSHA filter
-    if (PART_D_EVENT_CLAIM.equals(claimType)) {
-      expectMatch = false;
-    }
+    // PDE has no SAMHSA, so expect no match on SAMHSA filter
```

Comment:
I suspect the comment would be better above the initialization of the boolean.  That's more in keeping with its original placement.

Reply:
I did this and about 30 more....legacy/existing code that probably shouldn't have been touched in this PR